lxd/db/cluster/projects.go

```diff
@@ -236,3 +236,29 @@ func InitProjectWithoutImages(ctx context.Context, tx *sql.Tx, project string) e
 	_, err = tx.Exec(stmt, defaultProfileID)
 	return err
 }
+
+// GetAllProjectsConfig returns a map of project name to config map.
+func GetAllProjectsConfig(ctx context.Context, tx *sql.Tx) (map[string]map[string]string, error) {
```

Comment:
Can you use `dbCluster.GetConfig(ctx, tx.Tx(), "project")`

Reply:
Yeah it's not ideal for this use case unfortunately.